RtspServer/MediaTypes/RtpVideoSink.cs

```diff
@@ -2,12 +2,11 @@
 using Media.Common.Extensions.IPEndPoint;
 using Media.Common.Extensions.TimeSpan;
 using Media.Rtp;
-using Media.Rtsp.Server.MediaTypes;
-using Media.Sdp.Lines;
 using Media.Sdp;
+using Media.Sdp.Lines;
 using System;
-using static Media.Rtsp.Server.MediaTypes.RFC2435Media;
 using System.Collections.Generic;
+using static Media.Rtsp.Server.MediaTypes.RFC2435Media;
```

Comment:
The goal is eventually to remove RFC2435 Media from having to be used in a all the sinks, Lets just remove it now and remove this logic in SendPackets: 

```
                        
//Check for if previews should be updated (only for the jpeg type for now)
if (DecodeFrames && frame.PayloadType == RFC2435Frame.RtpJpegPayloadType)
{
    OnFrameDecoded((RFC2435Media.RFC2435Frame)frame);
}
```